tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1772,19 +1775,20 @@ void import_export_opaque_with_data( data_t *data, int type_arg,
     TEST_EQUAL( psa_get_key_type( &got_attributes ), type );
     if( attr_bits != 0 )
         TEST_EQUAL( attr_bits, psa_get_key_bits( &got_attributes ) );
+    else
+        TEST_EQUAL( (size_t) expected_bits, psa_get_key_bits( &got_attributes ) );
 
     /* Export the key */
     ASSERT_ALLOC( exported, export_size);
-    status = psa_export_key( key, exported, export_size, &exported_length );
-    TEST_EQUAL( status, PSA_SUCCESS );
-    if( status != PSA_SUCCESS )
-        goto exit;
+    PSA_ASSERT(psa_export_key( key, exported, export_size, &exported_length ));
 
     /* The exported length must be set by psa_export_key() to a value between 0
      * and export_size. On errors, the exported length must be 0. */
     TEST_ASSERT( exported_length != 0 );
-    TEST_ASSERT( status == PSA_SUCCESS || exported_length == 0 );
     ASSERT_COMPARE( data->x, data->len, exported, exported_length );
+
+    if( ! mbedtls_test_psa_exercise_key( key, usage_flag, 0 ) )
```

Comment:
`exercise_key` needs to be told which algorithm to use, and the usage flags have to allow this algorithm. Otherwise the only thing it can do is check that export works, which is already done above (in a more effective way because here we can also compare the export output with expected data).

Reply:
Archana points out that the test drivers don't implement any cryptographic operations yet.

So calling exercise_key here is pointless for now :frowning_face:  Please remove the call then, and add a comment that we should exercise the new key here but the test drivers don't support it yet.